templates/registration/register.html

```diff
@@ -8,25 +8,37 @@ <h2 class="extra-padding">Register</h2>
             <hr>
         </div>
         <div class="register-buttons">
+            <p class="register-copy">
+                Sign up with one click using Facebook
+            </p>
             <div class="button-container">
+                <a class="facebook-font" href="{% url 'social:begin' backend='facebook-register' %}">
                 <div class="sign-up-button facebook">
-                    <h5 class="button-font"><a class="facebook-font" href="{% url 'social:begin' backend='facebook-register' %}"><i class="fa fa-facebook button-icon-padding" aria-hidden="true"></i>Sign Up with Facebook</a></h5>
+                    <h5 class="button-font"><i class="fa fa-facebook button-icon-padding" aria-hidden="true"></i>Sign Up with Facebook</h5>
                 </div>
+                </a>
             </div>
             <div class="button-container">
+                <a class="google-font" href="{% url 'social:begin' backend='google-oauth2-register' %}">
                 <div class="sign-up-button google">
-                    <h5 class="button-font"><a class="google-font" href="{% url 'social:begin' backend='google-oauth2-register' %}"><i class="fa fa-google button-icon-padding" aria-hidden="true"></i>Sign Up with Google</a></h5>
+                    <h5 class="button-font"><i class="fa fa-google button-icon-padding" aria-hidden="true"></i>Sign Up with Google</h5>
                 </div>
+                </a>
             </div>
 
             <div class="button-container">
+                <a class="manual-font" data-open="signup-modal">
                 <div class="sign-up-button manually">
-                    <h5 class="button-font"><a class="manual-font" data-open="signup-modal"><i class="fa fa-envelope button-icon-padding" aria-hidden="true"></i>Sign Up with e-mail</a></h5>
+                    <h5 class="button-font"><i class="fa fa-envelope button-icon-padding" aria-hidden="true"></i>Sign Up with e-mail</h5>
                 </div>
+                </a>
             </div>
             <div class="reveal center" id="signup-modal" data-reveal>
                 <form action="" method="POST">
                     {% csrf_token %}
+                    <p class="color-grey">
+                        Please use all lowercase, no spaces, no special characters
```

Comment:
Passwords with lowercase characters, uppercase characters and special characters are stronger, so we should not discourage users from using those. Maybe instead we should advise to users that usernames and passwords _are_ case-sensitive.